### PR TITLE
Fixing HuggingFace LayerNorm.weight reference - replacing with PyTorch equivalents

### PR DESCRIPTION
**Pull request by michaellavelle**
A reference to "LayerNorm.weight" looks like it may have been ported from a HuggingFace implementation - when running locally the values that I need to exclude from weight decay in this implementation are "ln1.weight", "ln2.weight" and "ln_f.weight".

Update:  I've created another PR with a more generic fix for this issue:   https://github.com/karpathy/minGPT/pull/18

Thanks for putting out this awesome code - I'm working on a Java implementation of gpt and couldn't find any existing code to reference that contained both the sampling and training steps without introducing a lot of complexity or superfluous frameworks.   This is a really helpful project  - many thanks!

### CODE REVIEW

**Comment by michaellavelle** [comment]
Update: I've created another PR with a more generic fix for this issue: #18

**Comment by karpathy** [comment]
Fixed in https://github.com/karpathy/minGPT/commit/bbbdac74fa9b2e55574d70056163ffbae42310c1
